packages/editor/src/editor.spec.ts

```diff
@@ -83,6 +98,10 @@ describe('Editor', () => {
     });
   });
 
+  after(async() => {
+    await fs.rmdir(base, { recursive: true });
```

Comment:
Btw, my suggestion was about the `rmdir` call above, not about adding new `rmdir` calls :)

Reply:
Yeah, I was experimenting, it is still a draft pr ;) I think now it should be what you expect. Please let me know if it isn't.